### ISSUE TITLE
None of the injectors are working

### ISSUE THREAD

**Issue by kool601** [Wilee999/TTRInjector#4]
Title says it all... D:

**Comment by Wilee999** [comment]
aww

**Comment by kool601** [comment]
Will anyone fix them?

**Comment by ponyboy837** [comment]
Nope, Because Willie decided to be a dick and fuck with me. So go ahead and take em

**Comment by kool601** [comment]
Let's hope someone can fix them, I am not good with python, xD.

**Comment by ponyboy837** [comment]
I probably will be releasing a TTR hack, Not on this TTRInjector thing however, Willie ain't going to get my hacks.

**Comment by kool601** [comment]
Woop woop, I am excited to relive the days of injecting!

**Comment by ponyboy837** [comment]
Same here, It's about time to. When my hack get's out, I'll be making clones, Not just toon clones either... There will be clones of everything! EVEN COGS. Watch out TTR. Were coming.
